packages/tools/kadena-cli/src/utils/helpers.ts

```diff
@@ -113,9 +114,9 @@ export const passwordPromptTransform =
 
     const trimmedPassword = password.trim();
 
-    if (typeof passwordFile !== 'string' && trimmedPassword !== '') {
-      log.info(`You can use the ${flag} flag to provide a password.`);
-    }
+    // if (typeof passwordFile !== 'string' && trimmedPassword !== '') {
+    //   log.info(`You can use the ${flag} flag to provide a password.`);
+    // }
```

Comment:
what's the reason for commenting out this code? are we gonna revisit this later?